src/app/data/components/data-browser/data-browser.component.ts

```diff
@@ -1661,6 +1692,13 @@ export class DataBrowserComponent implements OnInit, OnDestroy {
     this.updateSearchParameters.refreshSearchParams(drillDown.getRawValue());
   }
 
+  processCommentsFilter(key: string, filterGroups: Object): void {
+    if (!this.fullSearch.has(key)) { this.displayOutdatedWarning = true; return; }
+    const comments: FormGroup = this.fullSearch.get(key) as FormGroup;
+    this.applyValuesToFormGroup(filterGroups[key], comments);
+    this.updateSearchParameters.refreshSearchParams(comments.getRawValue());
+  }
+
```

Comment:
is this now in both classes? (there is a processCommentsfilter in the class as well)